models/dataset.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 14 20:36:19 2021
+
+@author: NLEGENDN
+"""
+import os
```

Comment:
Imports order:
https://www.python.org/dev/peps/pep-0008/#imports

This is not too bad though, as we have bots on this repo that should correct the order of your imports automatically.